notebooks/signac_106_Ideal_Gas_Example.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>Make the <code>Note:</code>  larger here as well</p><p>(with tags/other fancy things we can make warnings like that render more like warnings and less like text)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
See above -- the notes render like this in Jupyter but not ReviewNB:

![image](https://user-images.githubusercontent.com/3943761/107470713-74147780-6b31-11eb-9ffc-599efe7d9044.png)